### PR TITLE
Enable workflow for csp-billing-adapter-google

### PR DESCRIPTION
**Pull request by brett060102**

### CODE REVIEW

File: tests/unit/test_plugin.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+missing_config_file = 'tests/data/config_missing.yaml'
+
+pm = get_plugin_manager()
+config = Config.load_from_file(
+    'tests/data/config_good.yaml',
+    pm.hook
+)
+
+
+def test_get_config_from_good_config_file():
```
Comment (rtamalin) [inline tests/unit/test_plugin.py]: So unless you are implementing a load_defaults() hook, this test is only exercising the core code and nothing from the google plugin.

File: tests/unit/test_plugin.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+from csp_billing_adapter.config import Config
+
+# Comment out the following for now, not currently used which
+# flake8 flags as error
+# import pytest
+# from unittest.mock import Mock, patch
+
+
+good_config_file = 'tests/data/config_good.yaml'
+bad_config_file = 'tests/data/config_bad.yaml'
```
Comment (rtamalin) [inline tests/unit/test_plugin.py]: You have only provided a config_good.yaml data file, but not the _bad or _missing variants...

**Comment by rtamalin** [comment]
Looks good in general, have a couple of questions.

File: tests/unit/test_plugin.py (hunk 2 above)
Comment (brett060102) [reply tests/unit/test_plugin.py]: Missing does not exist by design. I put the def in for bad really for future place for when there are actually tests.

File: tests/unit/test_plugin.py (hunk 1 above)
Comment (brett060102) [reply tests/unit/test_plugin.py]: Really only testing core. Since there is no real code yet, just trying to get basic test plumbing in place.

**Review by rtamalin — APPROVED** [review]

**Pull request merged**